includes/class-wp-job-manager-email-notifications.php

```diff
@@ -164,9 +166,8 @@ public static function lazy_init() {
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/emails/class-wp-job-manager-email-employer-expiring-job.php';
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/emails/class-wp-job-manager-email-admin-expiring-job.php';
 
-		if ( ! class_exists( 'Emogrifier' ) && class_exists( 'DOMDocument' ) && version_compare( PHP_VERSION, '5.5', '>=' ) ) {
-			include_once JOB_MANAGER_PLUGIN_DIR . '/lib/emogrifier/class-emogrifier.php';
-		}
+		// Load Vendor Autoload for Emogrifier.
```

Comment:
We don't need to be specific here, we could add more libraries in the future.

Reply:
Good point!  done in 856bbf44d8958098a46a392ccac4262bf60ecbf2